README.md

```diff
@@ -13,9 +13,14 @@ You will need a basic understanding of how to use the command line on your compu
 
 You will need Node and npm installed on your computer. Node is a JavaScript runtime used for developer tooling, and npm is the package manager included with Node. If you have a package manager installed for your operating system, setup can be as straightforward as:
 
-* macOS: `brew install node`
-* Windows: `choco install nodejs`
-* Ubuntu: `apt install nodejs npm`
+* macOS: `brew install node@14`
```

Comment:
Hmm, I don't think we should change these commands, especially since `choco` requires you to define a specific version. This will quickly become out of date.

I think just adding the note below that a specific version is required will be enough for the person to look up how to install specific versions of NodeJS using their preferred package manager.